examples/otlp-exporter-node/metrics.js

```diff
@@ -4,25 +4,50 @@ const { DiagConsoleLogger, DiagLogLevel, diag } = require('@opentelemetry/api');
 const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-http');
 // const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-grpc');
 // const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-proto');
-const { MeterProvider, PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics');
+// const { ConsoleMetricExporter } = require('@opentelemetry/sdk-metrics');
+const {
+  ExponentialHistogramAggregation,
+  MeterProvider,
+  PeriodicExportingMetricReader,
+  View,
+} = require('@opentelemetry/sdk-metrics');
 const { Resource } = require('@opentelemetry/resources');
-const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+const {
+  SemanticResourceAttributes,
+} = require('@opentelemetry/semantic-conventions');
 
 // Optional and only needed to see the internal diagnostic logging (during development)
 diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
 
-const metricExporter = new OTLPMetricExporter({});
+const metricExporter = new OTLPMetricExporter({
+  // headers: {
+  //   foo: 'bar'
+  // },
+});
+
+// Define view for the exponential histogram metric
+const expHistogramView = new View({
+  aggregation: new ExponentialHistogramAggregation(),
+  instrumentName: 'test_exponential_histogram',
```

Comment:
IMO, this comment better fits the option `instrumentName` in place.

```suggestion
  // Note, the instrumentName is the same as the name that has been passed for
  // the Meter#createHistogram function for exponentialHistogram.
  instrumentName: 'test_exponential_histogram',
```

Reply:
I believe I did what you suggested, let me know!